ravenframework/OutStreams/OutStreamInterface.py

```diff
@@ -54,7 +54,7 @@ def __init__(self):
     self.printTag = 'OutStreamInterface'
     self.overwrite = True       # overwrite existing creations?
     self.subDirectory = None    # directory to save generated files to
-    self.filename = ''          # target file name
+    self.filename = None        # target file name
```

Comment:
this is a better default to discern between case where the filename has been inputted and cases where has not